lib/ext/string.rb

```diff
@@ -0,0 +1,15 @@
+class String
```

Comment:
I think just having a formatter method that takes a string is maybe better than monkey patching a core class. If you do want to go this way though maybe look into refinements?